core-bundle/contao/dca/tl_preview_link.php

```diff
@@ -59,10 +59,7 @@
 			'delete',
 			'toggle',
 			'show',
-			'share' => array
-			(
-				'icon'                => 'share.svg'
-			)
+			'share' => array(),
```

Comment:
We should keep the existing icon and add a second variant with a checkmark instead of using two completely new icons. I will take care of creating the variant.

Reply:
I have added the icon in a5eaf733d5854dcb250a6301e955cfa0e5936a30.